app/frontend/styles/_pagination.scss

```diff
@@ -0,0 +1,134 @@
+// https://github.com/alphagov/govuk-frontend/blob/add-pagination-prototype/src/govuk/components/pagination/_index.scss
+.govuk-pagination {
```

Comment:
One last thing (promise!). We should be using the `app-*` namespace for this component, as it’s not part of govuk-frontend (yet). Did you use these styles: https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data-prototype/blob/main/app/components/pagination/_pagination.scss

Reply:
Yep, copied and pasted from there just changed `app-` to `govuk-` to match the markup but will do it the other way around then.

Reply:
https://design-system.service.gov.uk/get-started/extending-and-modifying-components/#use-a-unique-prefix-for-component-names